Source/SwiftyDropbox/Platform/SwiftyDropbox_macOS/OAuthDesktop.swift

```diff
@@ -7,9 +7,59 @@ import AppKit
 import WebKit
 
 extension DropboxClientsManager {
+    /// Starts a "token" flow.
+    ///
+    /// - Parameters:
+    ///     - sharedWorkspace: The shared NSWorkspace instance in your app.
+    ///     - controller: An NSViewController to present the auth flow from.
+    ///     - openURL: Handler to open a URL.
```

Comment:
thats a horrible parameter name for a completion block :(

Reply:
Agreed, since this interface is been used by developers for years, I'd rather not change the param naming today.